.swiftformat

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
1. white spaces at start of file?
2. please disable wrapMultilineStatementBraces